src/test/java/apicurito/tests/steps/OperationSteps.java

```diff
@@ -104,10 +104,17 @@ public void setParametersTypes(DataTable table) {       //TODO  add specific par
 
             if (Boolean.valueOf(dataRow.get(4))) {
                 OperationUtils.selectResponse(dataRow.get(5));
+                if (VersionUtils.is3x() && "media type".equalsIgnoreCase(dataRow.get(0))) {
```

Comment:
Extract creating to separate method too.
e.g. public void createMediaTypeForResponse(String mediaType, String response){...}